config/configgrpc/configgrpc_test.go

```diff
@@ -35,6 +35,50 @@ import (
 	"go.opentelemetry.io/collector/pdata/ptrace/ptraceotlp"
 )
 
+func TestNewDefaultKeepaliveClientConfig(t *testing.T) {
```

Comment:
This test will pass when a new default is added.  Instead of checking individual struct fields, the test should create a  KeepaliveClientConfig struct and manual set the expected values.  Then assert that the struct returned by `NewDefaultKeepaliveClientConfig` is equal.  The other tests in this file should follow the same pattern.  You can look at configtls_test.go as an example.

Reply:
should i remove the default values i set?